tests/test_text_processing_util_mds24.py

```diff
@@ -1,6 +1,9 @@
 import pytest
+import numpy as np
+from numpy.testing import assert_array_equal, assert_array_almost_equal
```

Comment:
Remove this. We can just add `np.testing.` before the assert functions. See below suggestions for examples.
```suggestion

```